draft-kazuho-httpbis-priority.md

```diff
@@ -549,6 +555,17 @@ reflecting the urgencies expressed in the requests, so that less urgent
 responses would receive less bandwidth in case the bottleneck exists between the
 server and the intermediary.
 
+### HTTP/1 origins {#http11}
+
+It is common in CDN infrastructure to support HTTP/2 or HTTP/3 at the client
+facing edge, but HTTP/1.1 to origin servers. Unlike with connection coalescing,
+the CDN will "de-mux" requests into discrete connections to the origin. As
```

Comment:
I would be more explanataory here. Soemthing like, 

"for HTTP versions that multiplex requests, such a CDN "demultiplexes"  requests by issuing the requests independently over one of more HTTP/1.1 connections.